eng/Versions.props

```diff
@@ -123,4 +123,10 @@
     <MicrosoftDiagnosticsMonitoringLibraryVersion>$(MicrosoftDiagnosticsMonitoringVersion)</MicrosoftDiagnosticsMonitoringLibraryVersion>
     <MicrosoftDiagnosticsMonitoringEventPipeLibraryVersion>$(MicrosoftDiagnosticsMonitoringEventPipeVersion)</MicrosoftDiagnosticsMonitoringEventPipeLibraryVersion>
   </PropertyGroup>
+  <PropertyGroup Label="Helix">
+    <!-- Ubuntu 18.04 requires older nodejs version due to glibc version -->
```

Comment:
Without understanding the full scope of this, would it potentially make more sense to make a newer version the default, and then reduce the version if needed (versus in this case, where it looks like we use the oldest version anything needs, and only bump up the version if needed)?

Reply:
Since all of Linux x64 queues (CBL Mariner 2.0, Debian 11, and Ubuntu 18.04) run from the same invocation, I can't change the version of what's installed on a per-queue basis; they all have the same correlation payload. It would require either installing multiple versions and having each queue dictate which version it will use OR refactor the project to make the groupings even more granular. I'm not sure which is better; I can log an issue for us to think about how we can move to newer Node installations even if we have to preserve v16 for Ubuntu 18.04.

Reply:
/todo Determine how to use newer nodejs version for most Helix queues